app/representers/api/podcast_representer.rb

```diff
@@ -75,11 +74,7 @@ class Api::PodcastRepresenter < Api::BaseRepresenter
     end
   end
 
-  link :series do
-    URI.join(cms_root, represented.prx_uri).to_s if represented.prx_uri
-  end
-
   link :account do
-    URI.join(cms_root, represented.prx_account_uri).to_s if represented.prx_account_uri
+    URI.join(id_root, represented.prx_account_uri).to_s if represented.prx_account_uri
```

Comment:
the prx auth integration uses access to id for looking up account info, so this is more correct